coka/file-bundle/3.0/config/packages/oka_file.yaml

```diff
@@ -4,4 +4,3 @@ oka_file:
         root_path: '%env(OKAFILE_STORAGE_ROOT_PATH)%'
         webserver:
             host: '%env(OKAFILE_STORAGE_WEB_SERVER_HOST)%'
-
```

Comment:
Why were these lines changed?

Reply:
Hi,
These lines have changed to allow you to use environment variables to configure where your files are stored and little like the doctrine URL database

Reply:
If I look at the diff. I see that you just removed an empty line in this file. 

Generally you should not modify other files or fix CS. I just asked if there is any particular reason for this change or if it just was “unrelated CS”.

Reply:
Well understood thank you